dvc/repo/get.py

```diff
@@ -17,6 +20,20 @@
 logger = logging.getLogger(__name__)
 
 
+def copy_git_file(repo, src, dst):
+    # Assert that the file does indeed exist in git before attempting to
+    # copy it.
+    if src not in repo.scm.repo.head.commit.tree:
+        raise FileOutsideRepoError(src)
```

Comment:
Not sure why we need this new exception. We are protecting ourselves against abspaths, right? We do support out-of-repo abspaths in regular DVC (as `dvc add` argument or `-o` or `-O` for `dvc run`), so maybe we would be better off by re-raising the NoOutputInExternalRepoError we got earlier if we see that `src` or if it is a relpath but there is no such path in our repo?

Reply:
The error message for NoOutputInExternalRepoError is
```
Output '{}' not found in target repository '{}'
```

if the wording Output isn't confusing for you I can re-use the exception NoOutputInExternalRepoError

considering that the command takes an output path, but that we are checking the input path I found the error message to be ambiguous and created a new one

Reply:
@danihodovic Good point! Especially since we now support not only outputs but regular git files too.